src/bin/super/util.rs

```diff
@@ -640,7 +640,8 @@ pub async fn create_ec2_instance(
     );
 
     let tags = vec![
-        Tag::builder().key("Name").value(swarm_name).build(),
+        Tag::builder().key("Name").value(format("swarm{}", swarm_number)).build(),
+        Tag::builder().key("UserAssignedName").value(swarm_name).build(),
```

Comment:
I don't know if this is what we want. But this means every new swarm including the ones created using the legacy approach will have the tag key of `UserAssignedName`